### PR TITLE
[Small] Update phdf to allow ghost zone plotting

### PR DESCRIPTION
**Pull request by lroberts36**
## PR Summary

This PR allows for offsetting blocks in plots so that ghost zones can also be plotted. Useful for debugging boundary communication and other issues. This should not change any behavior of phdf 

## PR Checklist

- [x] New features are documented.
- [x] Code is formatted
- [x] Changes are summarized in CHANGELOG.md
- [x] (@lanl.gov employees) Update copyright on changed files

### CODE REVIEW

**Comment by lroberts36** [comment]
@par-hermes format

File: scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py
Comment (Yurlungur) [inline scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py:68]: We're not always going to want this. Can this be behind a flag?

File: scripts/python/packages/parthenon_tools/parthenon_tools/phdf.py
Comment (Yurlungur) [inline scripts/python/packages/parthenon_tools/parthenon_tools/phdf.py:160]: I *think* understand what this means: you've grown each block to icnlude its ghost zones, and this requires "plotting" rather than physical coordinates, right?

**Review by Yurlungur — APPROVED** [review]
Approved assuming my comment below about block number is resolved.

File: scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py
Comment (lroberts36) [reply scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py:68]: By default, `block_ids` is an empty array so not passing it acts as a flag.

Comment (Yurlungur) [reply scripts/python/packages/parthenon_tools/parthenon_tools/movie2d.py:68]: :+1: works for me

File: scripts/python/packages/parthenon_tools/parthenon_tools/phdf.py
Comment (lroberts36) [reply scripts/python/packages/parthenon_tools/parthenon_tools/phdf.py:160]: yes, not really sure what the best nomenclature to use is but essentially you need to add space in the coordinate system between blocks so the ghost zones don't overlap.

Comment (Yurlungur) [reply scripts/python/packages/parthenon_tools/parthenon_tools/phdf.py:160]: Yeah I had trouble with this nomenclature, but I agree---I can't think of anything better.

**Comment by pgrete** [comment]
I just merged `develop` with the updated non-transpose IO format and had to fix a merge conflict (incl. adapt to the new format), see https://github.com/lanl/parthenon/pull/717/commits/192143548bef0f56f775e3a5afa7885685f7a03e
@lroberts36 assuming that you have a pipeline ready to test the plot with ghost zones, could you please confirm that the current version in this branch still works as expected (i.e., in other words that I didn't break things).

**Comment by lroberts36** [comment]
@pgrete: Yes, this still seems to work with the updated HDF5 output. Can you approve and merge?

**Comment by pgrete** [comment]
> @pgrete: Yes, this still seems to work with the updated HDF5 output. Can you approve and merge?

Yes, will do momentarily. I already reviewed yesterday but didn't approve yet waiting for confirmation that the merge didn't break things.

**Review by pgrete — APPROVED** [review]

**Pull request merged**
